Doppler.Currency.Test/CurrencyServiceTests.cs

```diff
@@ -146,5 +147,32 @@ public IEnumerator<object[]> GetEnumerator()
 
             IEnumerator IEnumerable.GetEnumerator() => GetEnumerator();
         }
+
+        private CurrencyHandler GetHandler(CurrencyCodeEnum currencyCode)
+        {
+            switch (currencyCode)
+            {
+                case CurrencyCodeEnum.Ars:
```

Comment:
It would be nice using the real factory in place of a new one, but it would imply mocking the service provider and probably it will be difficulter.